DFKPS/iocBoot/iocDFKPS-IOC-03/st.cmd

```diff
@@ -10,7 +10,6 @@ errlogInit2(65536, 256)
 
 epicsEnvSet "IOCNAME" "DFKPS_02"
```

Comment:
Changed

Regards,
Kathryn

From: Freddie Akeroyd [mailto:notifications@github.com]
Sent: 12 June 2016 21:08
To: ISISComputingGroup/EPICS-ioc
Cc: Baker, Kathryn (STFC,RAL,ISIS); Author
Subject: Re: [ISISComputingGroup/EPICS-ioc] Ticket1170 bug fixes (#27)


In DFKPS/iocBoot/iocDFKPS-IOC-03/st.cmd<https://github.com/ISISComputingGroup/EPICS-ioc/pull/27#discussion_r66726481>:

> @@ -10,7 +10,6 @@ errlogInit2(65536, 256)

>

>  epicsEnvSet "IOCNAME" "DFKPS_02"

Shouldn't this be DFKPS_03 ?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ISISComputingGroup/EPICS-ioc/pull/27/files/694b3e13ec823a9bfff67ad58da3b95654a4aa39#r66726481>, or mute the thread<https://github.com/notifications/unsubscribe/AKEEPVjKASTcIhCbprEdGnLIfkyW93-rks5qLGc0gaJpZM4IfOER>.